fireapp/lib/presentation/volunteer_home/volunteer_home_view_model.dart

```diff
@@ -33,7 +33,10 @@ class VolunteerHomeViewModel extends FireAppViewModel {
       }
       var shifts =
         await _shiftsRepository.getVolunteerShifts(userID);
-      _shifts.add(RequestState.success(shifts));
+      // Filter shifts where status is 'accepted'
+      var acceptedShifts = shifts.where((shift) => shift.status == 'accepted').toList();
```

Comment:
Please make "accepted" a constant. Just put it somewhere in the shift model file